### ISSUE TITLE
Adequate CLI to send storage

### ISSUE THREAD

**Issue by JulissaDantes** [recallnet/ipc#219]
Now that we are including storageAmount as part of the staking process a few changes must be done:
- For the cli join command the command must be able to get the storage amount argument and encode that as part of the metadata that we send to the contracts join function.
- For the stake command, we need to create users must be able to specify storageAmount, and get that argument value mapped out to the stake function call.
- For the unstake command we need to create a new unstake-storage that calls the unstakeStorage function in the subnetActorManagerFaucet.
- For the cli leave command nothing needs to be done.

In short: add stake and unstake commands for storage, and modify join command to include and send the storageAmount parameter as part of the metadata.
